### DISCUSSION TITLE
How blitz will help if I planning to use multiple clients in future?

### DISCUSSION THREAD

**Discussion by karthikcodes6** [blitz-js/blitz#1923] [Q&A] [answered]
Hi, I'm playing around with blitz for last few days and I have a question now. Since we are abstracting all the database, backend and frontend into one place. In case if I'm building a mobile app in future using the same backend how can I do this? How can I expose the API from blitz to use in the mobile application as well?

**Comment by flybayer** [comment, marked as answer]
See my answer here: https://github.com/blitz-js/blitz/discussions/1926
